docs/tutorials/troubleshooting/get-console-session.md

```diff
@@ -5,7 +5,7 @@ A console session can be helpful when VMs are not accessible through the normal
 Have edit or owner permissions on the namespace where the problematic VM is deployed. To confirm that you have the required permissions, run the following command:
 
 ```console
-$ kubectl auth can-i create webconsolerequests -n <namespace-name>
+$ kubectl auth can-i create virtualmachinewebconsolerequests -n <namespace-name>
```

Comment:
Because of the CRD name change the kubectl plugin still just always uses the v1a1 WebConsoleRequest